resources-for-our-team.html

```diff
@@ -18,6 +18,21 @@ <h4>Office Hours &amp; Tech Support</h4>
     <p>English Now!'s office is still open online and by phone - at regularly scheduled hours, Monday through Thursday, 9:00am until 5:00pm; Fridays 9:00am until 1:00pm; and Saturdays 9:30am until 12:00pm.</p>
     <p>Please note: 1) **Please call or text Ioanna on her mobile phone on Monday and Thursday between 2:00 and 4:00pm ONLY**; 2) if you are having a technical issue outside that time, please <u>text the school phone</u> at <a href="tel:+13017183575">+1.301.718.3575</a>. State succinctly what the issue is, and how urgent it is. We may change the protocol for tech support and will inform you if this changes. Thank you so much for your support!</p>
     <br>
+    <h4>Online Teaching: Zoom Protocols and Lessons Learned</h4>
+    <ol>
+        <li><strong>Zoom Basics: Setting Up Zoom and Joining a Class </strong>(For Instructors or Students) Download <a target="_blank" href="files/team-resources/English Now! - Setting Up and Joining a Class on Zoom.pdf">here</a>.</li>
+        <li><strong>Lessons Learned in Online Teaching-Week One. </strong>Download <a target="_blank" href="files/team-resources/English Now! - Lessons Learned in Online Teaching-Week One - 2020.03.20.pdf">here</a>. (thanks, Jennifer, for compiling this!)</li>
+    </ol>
+    <h4>Online Curriculum Resources (watch this space, this list will grow)</h4>
+    <ol>
```

Comment:
It strikes me that perhaps this should be an unordered list, although I'm guessing my dad gave you it as an ordered, so that's fine